app-backend/batch/src/main/scala/com/azavea/rf/batch/export/CheckExportStatus.scala

```diff
@@ -76,21 +76,28 @@ case class CheckExportStatus(exportId: UUID, statusURI: URI, time: Duration = 60
 
   def sendExportNotification(status: String, user: User, platform: Platform, name: Option[String], id: UUID, exportType: String) = {
     val email = new NotificationEmail
-    (user.emailNotifications, platform.publicSettings.emailExportNotification) match {
-      case (true, true) =>
+
+    val userEmailAddress: String = (user.emailNotifications, user.personalInfo.emailNotifications) match {
```

Comment:
I think this lives better on the `User` case class -- the same logic is duplicated across a few different files, this way if we need to update it later for some reason we are able to do so.